<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>相机模块</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>相机画面</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="camera_img">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="SAVE_PATH">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择存放路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MANUAL_SHOT">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>手动拍照</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AUTO_SHOT">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>自动拍照</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="STOP_CAMERA">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QRadioButton" name="rbtn1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Basler相机</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">CAMERA_SELECT</string>
           </attribute>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QRadioButton" name="rbtn2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>海康威视相机</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">CAMERA_SELECT</string>
           </attribute>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QRadioButton" name="radioButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>电脑相机</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">CAMERA_SELECT</string>
           </attribute>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="LAUNCH_CAMERA">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>启动相机</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>自动拍照间隔时间(ms)：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="INTERVAL">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="INTERVAL_SET">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>设定</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="CAMERA_SELECT">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
